web/models/package_metadata.ex

```diff
@@ -6,10 +6,11 @@ defmodule HexWeb.PackageMetadata do
     field :licenses, {:array, :string}
     field :links, :map
     field :maintainers, {:array, :string}
+    field :extra, :map
```

Comment:
We need to add this field to audit_log.ex also. https://github.com/hexpm/hex_web/blob/master/web/models/audit_log.ex#L38